test/prj/v15-seek-history.rzdb

```diff
@@ -0,0 +1,5847 @@
+/
+type=rizin rz-db project
+version=15
+
+/core
+blocksize=0x100
+offset=0x5af0
+
+/core/analysis
+
+/core/analysis/blocks
+
+/core/analysis/callables
+_Exit=func
```

Comment:
A lot of info that is not tested for in this file can be stripped away to reduce the file size. See for example `v7-zignatures.rzdb`, which contains almost only empty namespaces except where info is needed.